16:40:45.531 [qtp25763165-28] WARN  o.h.e.jdbc.spi.SqlExceptionHelper - SQL Error: 42102, SQLState: 42S02
16:40:45.531 [qtp25763165-28] ERROR o.h.e.jdbc.spi.SqlExceptionHelper - Table "TBL_USER" not found; SQL statement:
select user0_.id as id1_1_, user0_.login_name as login_na2_1_, user0_.name as name3_1_, user0_.password as password4_1_, user0_.register_date as register5_1_, user0_.roles as roles6_1_, user0_.salt as salt7_1_ from tbl_user user0_ where user0_.login_name=? [42102-173]
17:31:24.328 [main] WARN  o.s.aop.framework.CglibAopProxy - Unable to proxy method [public final void org.springframework.test.context.junit4.AbstractJUnit4SpringContextTests.setApplicationContext(org.springframework.context.ApplicationContext)] because it is final: All calls to this method via a proxy will be routed directly to the proxy.
17:31:24.640 [main] INFO  c.ilongfei.repository.JpaMappingTest - ok: User
17:31:24.656 [main] INFO  c.ilongfei.repository.JpaMappingTest - ok: Task
17:31:24.875 [main] WARN  o.s.aop.framework.CglibAopProxy - Unable to proxy method [public final void org.springframework.test.context.junit4.AbstractJUnit4SpringContextTests.setApplicationContext(org.springframework.context.ApplicationContext)] because it is final: All calls to this method via a proxy will be routed directly to the proxy.
17:31:25.234 [main] WARN  c.i.service.account.AccountService - 操作员foo尝试删除超级管理员用户
